sdk/identity/identity/Troubleshooting.md

```diff
@@ -137,41 +169,82 @@ The tenant ID is the Global Unique Identifier (GUID) that identifies your organi
-| [Azure Kubernetes Service](https://docs.microsoft.com/azure/aks/use-managed-identity)                                              | [Configuration Instructions](https://docs.microsoft.com/azure/aks/use-managed-identity)                                                                    |
-| [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/msi-authorization)                                                | Not Applicable                                                     |
-| [Azure Arc](https://docs.microsoft.com/azure/azure-arc/servers/managed-identity-authentication)                                    | [Configuration Instructions](https://docs.microsoft.com/azure/azure-arc/servers/security-overview#using-a-managed-identity-with-azure-arc-enabled-servers) |
-| [Azure Service Fabric](https://docs.microsoft.com/azure/service-fabric/concepts-managed-identity)                                  | [Configuration Instructions](https://docs.microsoft.com/azure/service-fabric/configure-existing-cluster-enable-managed-identity-token-service)             |
+If you have access to the VM, you can verify the manged identity endpoint is available via the command line using curl. 
+
+```bash
+curl 'http://169.254.169.254/metadata/identity/oauth2/token?resource=https://management.core.windows.net&api-version=2018-02-01' -H "Metadata: true"
+```
+> Note that output of this command will contain a valid access token, and SHOULD NOT BE SHARED to avoid compromising account security.
+### Azure App Service and Azure Functions Managed Identity
+
+#### CredentialUnavailableError
+| Error Message |Description| Mitigation |
+|---|---|---|
+|ManagedIdentityCredential authentication unavailable.|The environment variables configured by the App Services host weren’t present.|<ul><li>Ensure the managed identity has been properly configured on the App Service. Instructions for configuring the managed identity can be found [here](https://docs.microsoft.com/azure/app-service/overview-managed-identity?tabs=dotnet).</li><li>Verify the App Service environment is properly configured and the managed identity endpoint is available. See [below](#verifying-the-app-service-managed-identity-endpoint-is-available) for instructions.</li></ul>|
+#### __Verifying the App Service managed identity endpoint is available__
+
+If you have access to SSH into the App Service, you can verify managed identity is available in the environment. First ensure the environment variables `MSI_ENDPOINT` and `MSI_SECRET` have been set in the environment. Then you can verify the managed identity endpoint is available using curl.
+```bash
+curl 'http://169.254.169.254/metadata/identity/oauth2/token?resource=https://management.core.windows.net&api-version=2018-02-01' -H "Metadata: true"
+```
+> Note that the output of this command will contain a valid access token, and SHOULD NOT BE SHARED to avoid compromising account security.
```

Comment:
```suggestion

> Note that the output of this command will contain a valid access token, and SHOULD NOT BE SHARED to avoid compromising account security.
```